### DISCUSSION TITLE
Full-Width?

### DISCUSSION THREAD

**Discussion by SterlingSavariau** [pradyunsg/furo#365] [Ideas] [answered]
As much as I love hiding TOC on the index page, what if that's not enough? What if I would also like to have a full-width to stretch out the text content of my index? Is that something that wouldn't be too much to ask?

When I hide TOC, it looks like weird to have everything boxed when there's nothing but free space on the right.

**Comment by pradyunsg** [comment, marked as answer]
This is not possible currently.
